lib/page/static.js

```diff
@@ -4,6 +4,10 @@ var serveStatic = require('serve-static');
 
 module.exports = function(link, page, options){
   var router = page.router;
-  router.use(serveStatic( path.join(link.path,'www','public')));
+  router.use(serveStatic( path.join(link.path,'www','public'), {
+    setHeaders: function (res, path) {
+      res.header('Access-Control-Allow-Origin', '*');
```

Comment:
Hmm. While I'm not immediately aware of how this would be a security violation, I do think that opening up to all origins is wider than we need to be. Can we open up to just the required origin?

https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/providing-home-cards-for-the-amazon-alexa-app#image_hosting